.github/workflows/android-e2e.yml

```
@@ -0,0 +1,197 @@
+          echo "Installing $APK on $ANDROID_SERIAL"
+          adb install -r "$APK"
+
+      - name: Install Argent and start the tool-server
+        uses: ./.github/actions/argent-server
+        with:
+          simulator-server-log: debug
+
+      # `screenrecord` stops on its own after 3 minutes, so record in segments
+      # until the stop step interrupts it.
+      - name: Start screen recording
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:2:15: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)